docusaurus/docs/dev-docs/backend-customization/controllers.md

```diff
@@ -184,24 +198,30 @@ export default {
 When a new [content-type](/dev-docs/backend-customization/models#content-types) is created, Strapi builds a generic controller with placeholder code, ready to be customized.
 :::
 
-### Sanitization in controllers
+### Sanitization and Validation in controllers
 
 :::warning
-As of Strapi v4.8.0 and greater it's strongly recommended you sanitize your incoming request query and parameters utilizing the new `sanitizeQuery` function to prevent leaking of private data.
+It's strongly recommended you sanitize (v4.8.0) and/or validate (v4.13.0) your incoming request query and parameters utilizing the new `sanitizeQuery` and `validateQuery` functions to prevent leaking of private data.
```

Comment:
```suggestion
It's strongly recommended you sanitize (v4.8.0+) and/or validate (v4.13.0+) your incoming request query and parameters utilizing the new `sanitizeQuery` and `validateQuery` functions to prevent the leaking of private data.
```

I added "+" because I assumed you meant "with these versions and onwards". Do you know if my assumption was correct?